lib/src/dialogs/macos_alert_dialog.dart

```diff
@@ -176,19 +224,144 @@ class MacosAlertDialog extends StatelessWidget {
                       ],
                     ],
                   ),
-                ),
+                ],
               ],
+              const SizedBox(height: 16),
+              if (suppress != null)
+                DefaultTextStyle(
+                  style: MacosTheme.of(context).typography.headline,
+                  child: suppress!,
+                ),
+              const SizedBox(height: 16),
             ],
-            const SizedBox(height: 16),
-            if (suppress != null)
-              DefaultTextStyle(
-                style: MacosTheme.of(context).typography.headline,
-                child: suppress!,
-              ),
-            const SizedBox(height: 16),
-          ],
+          ),
         ),
       ),
     );
   }
 }
+
+/// Displays a macos alert dialog above the current contents of the app.
+Future<T?> showMacosAlertDialog<T>({
```

Comment:
Now that we have the `showMacosAlertDialog` function, it'd be good to remove the export to the material's `showDialog`.

Reply:
I couldn't find an export